stateproof/worker_test.go

```diff
@@ -857,3 +856,422 @@ func TestBuilderGeneratesValidStateProofTXN(t *testing.T) {
 
 	a.NoError(tx.Txn.WellFormed(transactions.SpecialAddresses{}, proto))
 }
+
```

Comment:
There is a log of code duplication in those tests. can you please try to create functions?
more to that, if there is no change in the protocol version, some of them could even be merged into one test that invokes `handleSig` multiple times.